source/users/custom-metadata.txt

```diff
@@ -107,6 +107,65 @@ collection. Depending on your use case, you may:
   standard CRUD operations from a Function, an Atlas Device SDK, a MongoDB
   driver, or MongoDB Compass.
 
+Permissions and Custom User Data
```

Comment:
```suggestion
Refresh User Permissions for Custom User Data
```
can't think of a better wording, but this is more about refreshing permissions, right?